src/MyEventEmitter.js

```diff
@@ -1,14 +1,72 @@
+/* eslint-disable max-len */
 'use strict';
 
 class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  constructor() {
+    this.events = {};
+  }
+
+  checkEventName = (eventName) => {
+    if (!this.events[eventName]) {
+      this.events[eventName] = [];
+    }
+  };
```

Comment:
The `checkEventName` method is a helper method and does not need to be a public method on the class. It would be better to make it a private method by using the `#` prefix, so it would look like this: `#checkEventName`. This way, it's clear that this method is not intended to be used outside of the class.